### ISSUE TITLE
Known issue: affects color of disabled elements.

### ISSUE THREAD

**Issue by teardownthiswall** [necolas/normalize.css#349]
I see this comment in the code but can't find an open issue related to this. 

Issue can bee seen on the UI test page, item 24 - text color on disabled buttons is still black.

http://necolas.github.io/normalize.css/latest/test.html

**Comment by indrekpaas** [comment]
Would it make sense to fix this with `:not()` selector?
```css
button:not(:disabled),
input:not(:disabled),
optgroup:not(:disabled),
select:not(:disabled),
textarea:not(:disabled) {
  color: inherit; /* 1 */
}

button,
input,
optgroup,
select,
textarea {
  font: inherit; /* 2 */
  margin: 0; /* 3 */
}
```

**Comment by gaopinghuang0** [comment]
I'd use this:
```css
button:disabled,
input:disabled,
optgroup:disabled,
select:disabled,
textarea:disabled {
	color: graytext;
}
```

**Comment by jonathantneal** [comment]
This has been resolved by #502.
